src/main/webapp/js/user-page-loader.js

```diff
@@ -90,9 +91,27 @@ function buildMessageDiv(message) {
   return messageDiv;
 }
 
-/** Fetches data and populates the UI of the page. */
+/** Fetches user data then adds it to the page */
+function fetchAboutMe(){
+  const url = '/about?user=' + parameterUsername;
+  fetch(url).then((response) => {
+    return response.text();
+  }).then((aboutMe) => {
+    const aboutMeContainer = document.getElementById('about-me-container');
+    if(aboutMe == ''){
+      aboutMe = 'This user has not entered any information yet.';
+    }
+    
+    aboutMeContainer.innerHTML = aboutMe;
+
+  });
+}
+
+// Fetches data and populates the UI of the page.
 function buildUI() {
   setPageTitle();
   showMessageFormIfViewingSelf();
   fetchMessages();
-}
\ No newline at end of file
+  fetchAboutMe();
```

Comment:
I would probably keep this in the same order it appears in the UI. So move this under setPageTitle()